### PR TITLE
 In TimetableItemDetailScreen, add animations to DescriptionSection

### PR DESCRIPTION
**Pull request by kosenda**
## Issue
- none

## Overview (Required)
- In the DescriptionSection, I thought it would be even better if the More button had an animation instead of just switching quickly.
- `ClickableTextExpandAnimateDurationMillis = 1000` may be a bit long, but I set it because I thought it would be richer UI and not disturb the user.

## Movie
Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/91f3b2ce-34a6-4a90-8c90-1065dbfcee6f" width="300" > | <video src="https://github.com/user-attachments/assets/61b69787-a17d-480a-98a4-e75fa9e58926" width="300" >

#### After and 5x slowed

https://github.com/user-attachments/assets/da33a610-9289-48c5-a3f2-a9b46df8f073

### CODE REVIEW

File: core/designsystem/src/commonMain/kotlin/io/github/droidkaigi/confsched/designsystem/component/ClickableLinkText.kt [hunk 1]
```diff
@@ -20,6 +22,8 @@ import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.text.style.TextOverflow
 
+private const val ClickableTextExpandAnimateDurationMillis = 1000
```
Comment (takahirom) [inline core/designsystem/src/commonMain/kotlin/io/github/droidkaigi/confsched/designsystem/component/ClickableLinkText.kt:25]: Thanks. ClickableTextExpandAnimateDurationMillis is a little lengthy.
Could we use the Medium duration for this?
https://m2.material.io/design/motion/speed.html#duration

**Review by takahirom — APPROVED** [review]
You nailed it! Thank you for your contribution!

**Comment by kosenda** [comment]
Here is the movie after modification!

https://github.com/user-attachments/assets/41046fc0-0483-4f64-9ab7-55de82144511

File: core/designsystem/src/commonMain/kotlin/io/github/droidkaigi/confsched/designsystem/component/ClickableLinkText.kt (hunk 1 above)
Comment (kosenda) [reply core/designsystem/src/commonMain/kotlin/io/github/droidkaigi/confsched/designsystem/component/ClickableLinkText.kt]: Thank you 🙇 I had not seen this page!

**Pull request merged**